src/Microsoft.IIS.Administration/Microsoft.IIS.Administration.csproj

```diff
@@ -106,6 +106,10 @@
     <Exec Command="powershell -NonInteractive -executionpolicy Unrestricted -command &quot;&amp;'..\..\scripts\build\Copy-Configs.ps1' '$(MSBuildProjectDirectory)'&quot;" />
   </Target>
 
+  <Target Name="CleanBuildDir" Condition="$(CleanBuildDir) != ''" AfterTargets="Build">
+    <Exec Command="powershell -NonInteractive -executionpolicy Unrestricted -command &quot;&amp;..\..\scripts\build\Clean-BuildDir.ps1&quot;" />
```

Comment:
Nit: recommend to use absolute path with $(SolutionDir) instead of a relative path for readability and in case the project file is moved to another location.